setup/pi/configure.sh

```diff
@@ -314,6 +333,17 @@ function configure_sns () {
     fi
 }
 
+function configure_discord () {
+    if [ ! -z "${discord_enabled+x}" ]
+    then
+        log_progress "Enabling Discord"
+        echo "export discord_enabled=true" > /root/.teslaCamDiscordSettings
```

Comment:
Unless I'm missing something, I don't think the "discord_enabled" variable needs to be defined in .teslaCamDiscordSettings

Reply:
"discord_enabled" is not necessary anywhere tbh, I just followed the other configure_<notification_option> functions. I will remove.